Firmware/RTK_Surveyor/Tasks.ino

```diff
@@ -819,6 +871,125 @@ void ButtonCheckTask(void *e)
         }
       }
     } //End Platform = RTK Facet
+    else if (productVariant == REFERENCE_STATION) //Check one momentary button
```

Comment:
I like this. Do you need to update your stickers to reflect the button function?

Reply:
Indeed. I think yes? Let's talk on Wednesday.

I'm even thinking about moving the BOOT switch out a little further, so it is proud. But leave the RESET switch flush - or slightly recessed... (I.e. force the user to reach for a handy pointy thing before pushing the RESET button...)

Reply:
Yes do that!